samples/bluetooth/mesh/sensor_server/src/model_handler.c

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/mesh/models.h>
+#include <dk_buttons_and_leds.h>
+#include "model_handler.h"
+
+#define TEMP_RANGE_COLUMNS_THRESHOLD 32
+
+/* The columns (temperature ranges) for relative
+ * runtime in a chip temperature
+ */
+static const struct bt_mesh_sensor_column columns[] = {
+	{ { 0 }, { TEMP_RANGE_COLUMNS_THRESHOLD } },
+	{ { TEMP_RANGE_COLUMNS_THRESHOLD }, { 100 } },
+};
```

Comment:
Could you try a few ranges that are likely to generate data in each under normal circumstances?

e.g. 0-20, 20-25, 25-30, 30-100?

It's okay to skip the threshold defines for this, I think

Reply:
Done